.changeset/heavy-hairs-join.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/accordion": major
```

Comment:
```suggestion
"@nextui-org/accordion": patch
```

Thanks! @outputboy please update the changeset to be a patch

Reply:
Thanks @jrgarciadev , updated.